GVFS/GVFS.Common/ProductUpgrader.cs

```diff
@@ -174,13 +174,20 @@ public virtual bool TryCreateToolsDirectory(out string upgraderToolPath, out str
                     catch (UnauthorizedAccessException e)
                     {
                         success = false;
-                        error = string.Join(Environment.NewLine, "Unauthorized access.", "Please retry gvfs upgrade from an elevated command prompt.", e.ToString());
+                        error = string.Join(
+                            Environment.NewLine, 
+                            "Unauthorized access.", 
+                            $"Make sure you have write permissions to directory {rootDirectoryPath} and run `gvfs upgrade` again.", 
+                            e.ToString());

```

Comment:
If a user will see this message should we include the exception callstack (`e.ToString()`)?

I've found it's not uncommon for users to see a callstack and assume GVFS crashed.